<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1583</width>
    <height>899</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Duphunter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="addFolder">
          <property name="text">
           <string>&amp;Add folder...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="filterLabel">
          <property name="text">
           <string>      &amp;Filter: </string>
          </property>
          <property name="buddy">
           <cstring>filter</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="filter"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTreeView" name="treeView">
        <property name="minimumSize">
         <size>
          <width>1000</width>
          <height>600</height>
         </size>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="selectOldest">
          <property name="text">
           <string>Select oldes&amp;t duplicates</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="selectNewest">
          <property name="text">
           <string>Select &amp;newest duplicates</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="invertSelection">
          <property name="text">
           <string>Invert selecti&amp;on</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="removeThis">
          <property name="text">
           <string>&amp;Remove selected</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="keepThis">
          <property name="text">
           <string>&amp;Keep selected</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="cancel">
          <property name="text">
           <string>&amp;Cancel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="execute">
          <property name="text">
           <string>&amp;Execute</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1583</width>
     <height>43</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
